Added Receive-Twilio-SMS workflow

mangatrai opened the pull request:
Added workflow to receive SMS from twilio gateway

mangatrai committed:
Initial Commit

mangatrai committed:
Edited to change and reflect names as Twilio

mangatrai committed:
Added PNG images

Added png images needed for Readme.md file

mangatrai committed:
Added export file of workflow

Added exported zip file of ReceiveMessageFromTwilio workflow

mangatrai committed:
removed phone number

mangatrai committed:
Updated ReadMe

mangatrai committed:
Updated Readme.md

changed hyperlink and description

peterlamar commented:
Looks good! Would you be able to submit the json extract from webmethods so that the reader can load the finished product and check their results?

mangatrai commented:
I did add json as zip file as import functionality require a zip file. https://github.com/mangatrai/webmethodsio-examples/blob/master/twilio-receive-sms/ReceiveMessageFromTwilio.zip

let me add json file as well.

thank you for the comments

mangatrai committed:
Added template json file

Added template JSON file ReceiveMessageFromTwilio.json to facilitate direct import of workflow into webMethods.io platform

peterlamar commented:
Brilliant, lets do both from now on
